tls/make-certs.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -xe
+
+docker build -t temporal_cfssl:test - < $(PWD)/tls/Dockerfile.cfssl
+mkdir .pki
+docker run --rm --workdir /pki -v $(PWD)/.pki:/pki -v $(PWD)/tls:/tls --entrypoint /tls/gen-certs.sh temporal_cfssl:test
+
+sudo chown root .pki/postgresql-key.pem && sudo chmod 640 .pki/postgresql-key.pem
```

Comment:
First, it's pretty unfortunate to have root-owned files in a user directory. Is there any way to avoid this, perhaps with the `--chown` flag to dockerfile `COPY`? Or doing a chown/chmod in a RUN after the COPY? What part actually requires the permissions?

If it's required the actual files in the volume be owned by root, how about doing the chown with another docker run (using busybox or something) to avoid the sudo?

Reply:
Unfortunately these files come in via bind mount and are not copied into the container images.
There are some worts to changing file permissions to bind mounts from inside containers.

I documented this step and made it a really simple script that this could be understood for the sake of this example.

Another possible alternative is to generate the cert material to a volume and mapping those to these containers.

Reply:
Moved to a named volume and only extracted the `ca.pem`.

Removes sudo completely.